Fix implicit declaration of functions and other header inclusion issues.
https://sourceforge.net/p/icns/code/ci/de0ae9ab28de0cd2d1e51a77d55ac03f5ba77f6a/
--- icnsutils/icns2png.c.orig
+++ icnsutils/icns2png.c
@@ -20,6 +20,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdint.h>
 #include <string.h>
 
 #include <getopt.h>
--- icnsutils/icontainer2icns.c.orig
+++ icnsutils/icontainer2icns.c
@@ -18,7 +18,6 @@
 Boston, MA 02110-1301, USA.
 */
 #include <stdio.h>
-#include <string.h>
 
 // --> find icns data[i]
 // --> data[i-9] - data[i-1] = "[######]" (id)
--- icnsutils/png2icns.c.orig
+++ icnsutils/png2icns.c
@@ -22,10 +22,13 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdint.h>
+#include <stdlib.h>
+#include <unistd.h>
 
 #include <errno.h>
 
 #include <png.h>
+#include <setjmp.h>
 #include <icns.h>
 
 #define	FALSE	0
--- src/icns.h.orig
+++ src/icns.h
@@ -20,13 +20,11 @@
 Boston, MA 02110-1301, USA.
 */
 
-#include <stdio.h>
-#include <stdlib.h>
-#include <stdint.h>
-#include <string.h>
-
 #ifndef _ICNS_H_
 #define	_ICNS_H_
+
+#include <stdio.h>
+#include <stdint.h>
 
 /* basic data types */
 typedef uint8_t         icns_bool_t;
--- src/icns_colormaps.h.orig
+++ src/icns_colormaps.h
@@ -22,6 +22,8 @@
 
 #ifndef _COLORMAPS_H_
 #define	_COLORMAPS_H_	1
+
+#include "icns.h"
 
 typedef struct icns_colormap_rgb_t
 {
